src/Compilers/CSharp/Test/Symbol/Symbols/Source/PropertyTests.cs

```diff
@@ -288,13 +288,7 @@ public void AutoNoGet()
 }";
             var comp = CreateCompilation(text, parseOptions: TestOptions.Regular);
 
-            comp.VerifyDiagnostics(
-// (4,20): error CS8051: Auto-implemented properties must have get accessors.
-//     public int Q { set; } = 0;
-Diagnostic(ErrorCode.ERR_AutoPropertyMustHaveGetAccessor, "set").WithArguments("C.Q.set").WithLocation(4, 20),
-// (5,20): error CS8051: Auto-implemented properties must have get accessors.
-//     public int R { set; }
-Diagnostic(ErrorCode.ERR_AutoPropertyMustHaveGetAccessor, "set").WithArguments("C.R.set").WithLocation(5, 20));
+            comp.VerifyDiagnostics();
```

Comment:
>comp.VerifyDiagnostics(); [](http://example.com/codeflow?start=12&length=25)

Removal of this diagnostics feels unexpected. Could you elaborate, please?

Reply:
>  I was following the spec, which had an unintended change.

I am not sure how this was following the spec. The spec used to say: "An auto-property is not allowed to only have a single semicolon-only `set` accessor without a `get` accessor.``` Based on that, an error should be reported, I think.

Reply:
Oh yeah sorry. The spec fix is about `public int R { set => field; }`
I was tracking the case for `set;` which I mistakenly dropped in a TODO list I have locally :)